build.sbt

```diff
@@ -35,8 +35,6 @@ enablePlugins(
   UnidocRoot,
   UnidocWithPrValidation,
   NoPublish,
-  CopyrightHeader,
-  CopyrightHeaderInPr,
   ScalafixIgnoreFilePlugin,
```

Comment:
So these two are enabled one by one now?

Reply:
`CopyrightHeaderInPr` is removed in this PR since it was pointless and `CopyrightHeader` is now an autoPlugin that is by default enabled.